.github/workflows/raedar_tests CI/raedar.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: raedar tests
+
+on:
+  push:
+    branches: [ master,develop ]
```

Comment:
Remove master, for now, leave `develop`

Reply:
for array variables, I'd rather you use the hyphen when specifying multiple variables instead of the brackets. the hyphen is more acceptable around the programming world.
For example:
```
    branches:
        - develop
        - master
```

Reply:
for now though, only trigger the job on the develop branch.

Reply:
I have only set it for develop branch